Class/Tools.php

```diff
@@ -391,7 +512,15 @@ public static function htmlRadio($name, $input, $value = null, $options = array(
         return mb_substr($result, mb_strlen(@$options['separator']));
```

Comment:
Tím @ si v celé téhle metodě koleduješ o nevystopovatelné problémy.

Reply:
Dobrá připomínka - upravil jsem to tak, aby $options['separator'] (a další indexy) byly definované vždy.